### PR TITLE
Alex k ino/added ProgressPie to StoryBook

### LINKED ISSUE #23: Create a `ProgressPie` component and add to Storybook
**What needs to be done? Why does it need to be done?**

The component should consist of text inside of a pie chart. The Pie slices' color & width should be configurable via props. The text color and contents should be configurable via props.

Screen Shot 2020-12-23 at 3 12 47 PM
Screen Shot 2020-12-23 at 3 12 49 PM
Screen Shot 2020-12-23 at 3 12 52 PM
Screen Shot 2020-12-23 at 3 13 01 PM

Additionally, we should have example props for each of the states specified in the mocks (0,1,2,3,4,5,6,7)

### PR DESCRIPTION
**Pull request by AlexK-ino**
## Description of the change

ProgressBar has been moved to a separate component and added to the storybook

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #23

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/styles/ProgressPie.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+import styled from "styled-components";
+import ProgressBar from "react-customizable-progressbar";
```
Comment (ohaibbq) [inline src/styles/ProgressPie.styles.tsx:18]: This module was not added to our `package.json`

**Review by ohaibbq — CHANGES_REQUESTED** [review]

File: src/styles/ProgressPie.styles.tsx (hunk 1 above)
Comment (AlexK-ino) [reply src/styles/ProgressPie.styles.tsx:18]: Hello, Dan!

Perhaps I didn't quite get the point. To clarify:
My package.json has this module. Or is it not meant to use third-party modules?
Снимок экрана 2020-12-30 в 12 46 59

File: .eslintrc.js [hunk 2]
```diff
@@ -14,6 +14,9 @@ module.exports = {
     "import/resolver": { node: { extensions: [".js", ".jsx", ".ts", ".tsx"] } },
   },
   rules: {
+    "object-curly-newline": "off",
+    "react/jsx-curly-newline": "off",
+    "react/jsx-one-expression-per-line": "off",
```
[the remark this answers is not in the document]
Comment (AlexK-ino) [reply .eslintrc.js:19]: Снимок экрана 2020-12-30 в 13 56 56

These rules do not violate the `airbnb` style, with the exception of `"object-curly-newline",` which conflicts with prettier

File: src/styles/ProgressPie.styles.tsx (hunk 1 above)
Comment (AlexK-ino) [reply src/styles/ProgressPie.styles.tsx:18]: Dan, сould you give an explanation? Maybe I misunderstood?

Comment (ohaibbq) [reply src/styles/ProgressPie.styles.tsx:18]: I must have had an issue with my local environment. It works fine now!

**Review by ohaibbq — APPROVED** [review]

**Pull request merged**